engine/runtime-integration-tests/src/test/java/org/enso/interpreter/test/MetaObjectTest.java

```diff
@@ -272,6 +274,33 @@ public void numbersAreEitherIntegerOrFloat() throws Exception {
     }
   }
 
+  /**
+   * Primitive values and exceptions currently don't have an associated language.
+   *
+   * <p>TODO[PM]: Will be implemented in https://github.com/enso-org/enso/pull/11468
```

Comment:
How do you want to associate a language with `java.lang.Long` or `java.lang.Boolean`? What language you'd like to associate with such values? JavaScript? Enso?

Language can only be associated with Enso objects - e.g. subtypes of `EnsoObject`!